.github/workflows/link-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown links

```

Comment:
I can't comment on the above, but the issue templates are empty. If you're going to add those, you should fill them in. I just copy from ppn repo and switch out the product name.